### PR TITLE
Add new expectation: toRespondWithMimeType.

### PR DESCRIPTION
**Pull request by amyboyd**

### CODE REVIEW

File: app/Resources/docs/expectations/to_respond_with_mime_type.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+The _ToRespondWithMimeType_ expectation expects the URL given as the actual value to respond with the mime type given as the expected value. The expected value is required. The actual value should be a valid URL.
+
+## Example
+
+```
+Expect https://www.google.co.uk/ toRespondWithMimeType text/html
+```
+
+This expectation will check that https://www.google.co.uk/ responds with the mime typr "text/html".
```
Comment (zsturgess) [inline app/Resources/docs/expectations/to_respond_with_mime_type.md]: A little typo here :)

Comment (amyboyd) [inline app/Resources/docs/expectations/to_respond_with_mime_type.md]: @zsturgess Fixed!

Comment (zsturgess) [inline app/Resources/docs/expectations/to_respond_with_mime_type.md]: I... hadn't even finished reading the rest of the diff :+1:
I'll merge this when it passes.

**Comment by zsturgess** [comment]
This has only failed Scrutinizer because of the extra checks I've enabled for https://github.com/zsturgess/overwatch/issues/104

Merging anyway.

**Pull request merged**